src/components/LmsConfigurations/index.test.jsx

```diff
@@ -39,37 +47,85 @@ REQUIRED_CANVAS_CONFIG_FIELDS.forEach((field) => {
   canvasResponse.data.results[0][field] = 'testdata';
 });
 
+REQUIRED_BLACKBOARD_CONFIG_FIELDS.forEach((field) => {
+  formData.append(field, 'testdata');
+  blackboardResponse.data.results[0][field] = 'testdata';
+});
+
 const waitForAsync = () => new Promise(resolve => setImmediate(resolve));
 
 describe('<LmsConfigurations /> ', () => {
-  it('get LMS configurations when present', async () => {
+  it('get the Moodle LMS configuration when present', async () => {
     LmsApiService.fetchMoodleConfig.mockResolvedValue(moodleResponse);
-    LmsApiService.fetchCanvasConfig.mockResolvedValue(canvasResponse);
+
+    // Mock the fetch of the other configs to 404, not found
+    LmsApiService.fetchBlackboardConfig.mockRejectedValue(notFoundBlackboardResponse);
+    LmsApiService.fetchCanvasConfig.mockRejectedValue(notFoundCanvasResponse);
```

Comment:
So, not 100% necessary for now, but it would be nice, if we're repeating these calls for each test, to move these into a `beforeEach` function. (That being said, I've had some trouble with that with Jest, so, if you try it and it doesn't work, go back to this.)